code/go/0chain.net/chaincore/chain/state_sync.go

```diff
@@ -214,6 +245,35 @@ func (c *Chain) getStateNodes(ctx context.Context, keys []util.Key) (*state.Node
 	return ns, nil
 }
 
+func (c *Chain) getStateNodesFromSharders(ctx context.Context, keys []util.Key) (*state.Nodes, error) {
+	nsRequestor := StateNodesRequestor
+	params := &url.Values{}
+	for _, key := range keys {
+		params.Add("nodes", util.ToHex(key))
+	}
+	ctx, cancelf := context.WithCancel(common.GetRootContext())
+	var ns *state.Nodes
+	handler := func(ctx context.Context, entity datastore.Entity) (interface{}, error) {
+		rns, ok := entity.(*state.Nodes)
```

Comment:
If "r" means "result" in `rns`, maybe just call variable "result"?

Reply:
We should encourage cleaner style.. All those abbreviations really spend time when you read code, especially for new people